examples/cryptocurrency-advanced/backend/tests/api.rs

```diff
@@ -264,6 +268,55 @@ fn test_unknown_wallet_request() {
     api.assert_no_wallet(tx.author());
 }
 
+#[test]
+fn test_config_endpoint() {
+    let (_, api) = create_testkit();
+    let config = api.get_config().unwrap();
+    assert_eq!(config.init_balance, SERVICE_CONFIG.init_balance)
+}
+
+#[test]
+fn test_change_configuration() {
+    let (mut testkit, api) = create_testkit();
+    // Create and send a transaction via API
+    let (tx, _) = api.create_wallet(ALICE_NAME);
+    testkit.create_block();
+    api.assert_tx_status(tx.object_hash(), &json!({ "type": "success" }));
+
+    // Check that the user indeed is persisted by the service.
+    let wallet = api.get_wallet(tx.author()).unwrap();
+    assert_eq!(wallet.owner, author_address(&tx));
+    assert_eq!(wallet.name, ALICE_NAME);
+    assert_eq!(wallet.balance, 100);
+
+    // Change an initial balance to 200 via changing configuration.
+    let tx = api.change_config(&mut testkit, Config { init_balance: 200 });
+    testkit.create_block();
```

Comment:
Wouldn't it be simpler to use an existing API designed for this purpose:

```rust
let block = testkit.create_block();

assert!(block[tx.object_hash()].status().is_ok())
```

?

Reply:
To be fair, these are HTTP API tests, and `assert_tx_status` method is already used throughout the code.

Reply:
Then I think the approach should be unified, since suggested approach is already used in this PR: https://github.com/exonum/exonum/pull/1760/files#diff-480f4bde234f6ab105cb6acf68052b6aR315